proto/skaffold.proto

```diff
@@ -567,10 +602,19 @@ enum SuggestionCode {
     GCLOUD_DOCKER_AUTH_CONFIGURE = 103;
     // Run docker auth configure
     DOCKER_AUTH_CONFIGURE = 104;
-    //Verify Gcloud Project
+    // Verify Gcloud Project
     CHECK_GCLOUD_PROJECT = 105;
     // Check if docker is running
     CHECK_DOCKER_RUNNING = 106;
+    // Fix User Build Error
+    FIX_USER_BUILD_ERR = 107;
+    // Docker build internal error, try again
+    DOCKER_BUILD_RETRY = 108;
+    // Fix `cacheFrom` config for given artifact and try again
+    FIX_CACHE_FROM_ARTIFACT_CONFIG = 109;
+    // Fix `dockerfile` config for a given artifact and try again.
+    FIX_DOCKERFILE_CONFIG = 110;
```

Comment:
This name did make me think it was about the Dockerfile content rather than the `skaffold.yaml`'s docker configuration.  Would it make sense to have a class of fixes like `FIX_SKAFFOLD_CONFIG_DOCKERFILE`?

Reply:
done